config/default/manager_auth_proxy_patch.yaml

```diff
@@ -63,6 +63,7 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
+        - "--zap-log-level={{max (.Values.verbosity | default 2 | int) 1}}"
```

Comment:
The defaulting here is new behvior, which should be documented in values.yaml and `docs/users.md`. Does not have to be done in this PR.